### ISSUE TITLE
Android issue

### ISSUE THREAD

**Issue by valeriopisapia** [Heydon/beadz-drum-machine#5]
Nice work man :)
There is an issue on Android devices.
Basically, if I change the beat of a sound quickly, I'm getting a clipping issue.
I tried to fix it but I didn't find any good solution so far.
Any idea?

**Comment by Heydon** [comment]
Hi! The beat of a sound as in the tempo?

**Comment by valeriopisapia** [comment]
Hi @Heydon,
I'm sending you a video in order to explain it easily :).
Quick notes:
- it works on iOS devices perfectly (all browsers).
- That issue is only on Android devices.

[video.mp4.zip](https://github.com/Heydon/beadz-drum-machine/files/2521755/video.mp4.zip)

Hoping it could be helpful in order to find the issue.

**Comment by Heydon** [comment]
@valeriopisapia That;s very kind. I'll try to take a look.

**Comment by valeriopisapia** [comment]
@Heydon Did you take a look on it? found something?

**Comment by Heydon** [comment]
@valeriopisapia Yes, thanks. I don't think it's a specific issue so much as a generic performance problem. It works well on my Android, but I've never tried to push it quite that hard! Eventually, I will probably try a rewrite and hopefully I can improve performance as part of that.

**Comment by valeriopisapia** [comment]
@Heydon I tried to improve the performance (I will show you my version then if you want :) ) but I didn't find the right way :(
